vizro-core/src/vizro/actions/_callback_mapping/_get_action_callback_mapping.py

```diff
@@ -18,7 +18,9 @@
 from vizro.managers._model_manager import ModelID
 
 
-def _get_action_callback_mapping(action_id: ModelID, argument: str) -> Union[Dict[str, Union[Input, State, Output]]]:
+def _get_action_callback_mapping(
+    action_id: ModelID, argument: str
+) -> Union[List[dcc.Download], Dict[str, Union[Input, State, Output]]]:
```

Comment:
All of `Union[Input, State, Output]` are just subclasses of `DashDependency`, so you could just make it that if you prefer.

Or if you think it's clearer this way then fine, up to you 🙂